server/sonar-web/src/main/js/apps/issues/sidebar/LanguageFacet.js

```diff
@@ -38,17 +39,18 @@ type Props = {|
   referencedLanguages: { [string]: ReferencedLanguage },
   languages: Array<string>
 |};
+*/
 
 export default class LanguageFacet extends React.PureComponent {
-  props: Props;
+  /*:: props: Props; */
 
   static defaultProps = {
     open: true
   };
 
   property = 'languages';
 
-  handleItemClick = (itemValue: string) => {
+  handleItemClick = (itemValue /*: string */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)